dlrs/main/classes/ManageLookupRollupSummariesController.cls

```diff
@@ -156,6 +174,50 @@ public with sharing class ManageLookupRollupSummariesController {
     return null;
   }
 
+  public void hasChildTriggers() {
+    if (LookupRollupSummary == null) {
+      return;
+    } else {
```

Comment:
no else needed; thats the charm of early returns :)

Reply:
Yeah that's obvious looking back on it lol.